### ISSUE TITLE
Sample: type text can be an html string

### ISSUE THREAD

**Issue by sKY02644** [Hubertformin/electron-pos-printer#101]
Kindly give a sample how to / type text can be an html string /

or how can i align two text on a row with spacebetween

**Comment by unaisshemim** [comment]
yes i need to know wheather how can we pass the HTML content to data because it is easy for styling

**Comment by Hubertformin** [comment]
Confirmed supported — type text accepts a full HTML string. Example: { type: 'text', value: '<b>Hello</b> <span style="color:red">World</span>' }
